.gitignore

```diff
@@ -27,6 +27,7 @@
 
 # PyCharm files
 /.idea/
+*.iml
```

Comment:
Not sure if `.iml` files are that common. I'd rather have them not included in `.gitignore`.